src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -927,24 +894,12 @@ def open_dialog(self, slack_request: SlackActionRequest, group: Group) -> None:
             else:
                 self._update_modal(slack_client, external_id, modal_payload, slack_request)
 
-            metrics.incr(
-                SLACK_WEBHOOK_GROUP_ACTIONS_SUCCESS_DATADOG_METRIC,
-                sample_rate=1.0,
-                tags={"type": f"{self.dialog_type}_modal_open"},
-            )
         except SlackApiError:
-            metrics.incr(
-                SLACK_WEBHOOK_GROUP_ACTIONS_FAILURE_DATADOG_METRIC,
-                sample_rate=1.0,
-                tags={"type": f"{self.dialog_type}_modal_open"},
-            )
-            _logger.exception(
+            _logger.info(
                 "slack.action.response-error",
```

Comment:
for here and in the other `logger.info -`>` logger.exception`, would it be helpful to add the `exc_info` or report the error to sentry?